### PR TITLE
refactor(webhook): rename webhooks to webhook events

### PR DESCRIPTION
**Pull request by balzdur**
Small PR to rename our internal `webhooks` table and entities to `webhook_event`.

Better naming to distinguish with future operation on managing actual webhooks (endpoints + subscription in Convoy representation)

### CODE REVIEW

File: repositories/migrations/20240708141735_create_webhook_table.sql [hunk 1]
```diff
@@ -2,7 +2,7 @@
 -- +goose StatementBegin
 
 CREATE TABLE IF NOT EXISTS
-      webhooks (
+      webhook_events (
```
Comment (Pascal-Delange) [inline repositories/migrations/20240708141735_create_webhook_table.sql:5]: remember to make sure to run it locally so you don't deviate

**Review by Pascal-Delange — APPROVED** [review]
👍

File: repositories/migrations/20240708141735_create_webhook_table.sql (hunk 1 above)
Comment (balzdur) [reply repositories/migrations/20240708141735_create_webhook_table.sql:5]: Everytime I modify this file I run locally a `goose down` + `goose up` to check everything is clean

File: repositories/webhook_events_repository.go [hunk 2]
```diff
@@ -43,7 +43,7 @@ func (repo MarbleDbRepository) ListWebhookEvents(ctx context.Context, exec Execu
 		query = query.Where(squirrel.Eq{"delivery_status": mergedFilters.DeliveryStatus})
 	}
 
-	query = query.OrderBy("created_at DESC").Limit(mergedFilters.Limit)
+	query = query.Limit(mergedFilters.Limit)
```
Comment (Pascal-Delange) [inline repositories/webhook_events_repository.go:46]: for what it's worth, you can add it directly at the end of line 40 (with the query builder, order is not relevant)

**Review by Pascal-Delange — APPROVED** [review]

**Pull request merged**